### ISSUE TITLE
Handle swapping more similarly to BotW & TotK

### ISSUE THREAD

**Issue by TheCatMan2000** [TrueHorse/Wild-Tool-Access#17]
First installing this mod, I was a bit confused how to swap tools because holding the R key just flickered the menu.
I discovered the hard way that I needed to press the R key, swap to the tool or weapon, then LClick to select it. I nearly killed one of my tamed wolves swapping this way.

I expected to have to hold the R key, swap to the desired tool or weapon, then release to select, similar to how BotW and TotK handle weapon/shield/bow swaps.

**Comment by TrueHorse** [comment]
Thank you for taking the time to give your Feedback. :)
The attacking issue has been fixed in 1.3.0, which I just released.
Regarding the  disparity to BotW, I think I didn't do it that way because this was my first mod and it was more straight forward to just add a normal key binding. I will consider adding it in the future, but having finished 1.3.0, I will probably not work on this mod for a little bit.
